notifyall/notifyall.php

```diff
@@ -46,7 +47,7 @@ function notifyall_post(&$a)
 
 	$sitename = $a->config['sitename'];
 
-	if (!x($a->config['admin_name']))
+	if (empty($a->config['admin_name']))
 	{
 		$sender_name = '"' . L10n::t('%s Administrator', $sitename) . '"';
 	}	else 	{
```

Comment:
Uh? To much spaces here?

Reply:
Looks like tabs, could have looked like spaces with the right tab setting.

Reply:
Its tabs where spaces should have been.